src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -656,14 +652,6 @@ public ChainedHeader FindHeaderAndVerifyBlockIntegrity(Block block)
                 throw new BlockDownloadedForMissingChainedHeaderException();
```

Comment:
Probably should change it to return null instead of throwing exception and allow caller handle it